python_src/src/lamp_py/postgres/rail_performance_manager_schema.py

```diff
@@ -177,7 +177,7 @@ class TempEventCompare(SqlBase):  # pylint: disable=too-few-public-methods
     )
 
 
-class MetadataLog(SqlBase):  # pylint: disable=too-few-public-methods
+class LegacyMetadataLog(RpmSqlBase):  # pylint: disable=too-few-public-methods
```

Comment:
Should we just be dropping this table?

Reply:
i think that depends on the deployment order.

we'll discuss in a call.